Package/RedPy/redpy.py

```diff
@@ -63,109 +49,42 @@ def _DownloadFiles(self, jsonfile, number_of_files):
             print("Error creating folder")
             return
 
-        index = 0 #used to name the files
+        index = 0  # used to name the files
         for image_link in image_links:
             image_link = image_link.replace('amp;', '')
             f = _requests.get(image_link)
-
-            if f.status_code==200:
-                media_file = open(f'{_os.getcwd()}/red_media/{index}.jpg', 'wb')
+            if f.status_code == 200:
+                media_file = open(_os.path.join(
+                    _os.getcwd(), "red_media", f"{index}.jpg"), 'wb')
 
                 for chunk in f.iter_content(100000):
                     media_file.write(chunk)
                 media_file.close()
                 print("Downloaded")
-                index+=1
+                index += 1
         print("Download complete")
         global flag
-        flag=1
-
-    def download_videos(self, subreddit, number=5, sort_option=None):
-        """Downloads images from subreddit.
-            subreddit="Name of subreddit"
-            number=Number of images to be downloaded
-            sort_option=new/hot/top
-        """
-        subreddit = subreddit.strip('/')
-        if sort_option == None:
-            sort_option = ''
-
-        self.url = 'https://www.reddit.com/r/' + subreddit + '/'+  sort_option  +  '.json'
-        print(self.url)
-
-        user = {'user-agent':self.user}
-        res = _requests.get(self.url, headers=user)
-
-        if res.status_code != 200:
-            print("Could not download")
-            print(res.status_code)
-            return
-        self._DownloadVideoFiles(res.json(), number)
-        
-
-    def _DownloadVideoFiles(self, jsonfile, number_of_files):
-        video_links = self._getVideos(jsonfile, number_of_files)
-
-        if not self.createFolder():
-            print("Error creating folder")
-            return
-
-        index = 0 #used to name the files
-        for video_link in video_links:
-            video_link = video_link.replace('amp;', '')
-            f = _requests.get(video_link)
-
-            if f.status_code==200:
-                media_file = open(f'{_os.getcwd()}/red_media/{index}.mp4', 'wb')
-
-                for chunk in f.iter_content(100000):
-                    media_file.write(chunk)
-                media_file.close()
-                print("Downloaded")
-                index+=1
-        print("Download complete")
-        global flag
-        flag=1
-        
-        
-        
-    def _getVideos(self, jsonfile, number_of_files):
-        videos = []
-
-        for i in range(number_of_files):
-            try:
-                videos.append(jsonfile['data']['children'][i]['data']['preview']['reddit_video_preview']['fallback_url'])
-            except Exception as e:
-                print(e)
-        return videos
+        flag = 1
```

Comment:
This flag hasn't been defined anywhere in your code

Reply:
Yeah I added missing video download part  
And removed flag